src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -88,6 +95,22 @@ public void authorConfig_noSpecialCharacter_success() throws IOException, Invali
         Assert.assertEquals(AUTHOR_CONFIG_NO_SPECIAL_CHARACTER_AUTHORS, config.getAuthorList());
     }
 
+    @Test
+    public void authorConfig_emptyLocation_success() throws ParseException, IOException {
+        RepoConfiguration expectedConfig = new RepoConfiguration(new RepoLocation(""));
+
+        AuthorConfigCsvParser authorConfigCsvParser =
+                new AuthorConfigCsvParser(AUTHOR_CONFIG_EMPTY_LOCATION_FILE);
+        List<RepoConfiguration> authorConfigs = authorConfigCsvParser.parse();
+        RepoConfiguration authorConfig = authorConfigs.get(0);
+
+        Assert.assertEquals(1, authorConfigs.size());
+        Assert.assertEquals(expectedConfig, authorConfig);
+        Assert.assertEquals(expectedConfig.getLocation(), authorConfig.getLocation());
+        Assert.assertEquals(expectedConfig.getBranch(), authorConfig.getBranch());
```

Comment:
`Assert.assertEquals(expectedConfig.getLocation(), authorConfig.getLocation());` and `Assert.assertEquals(expectedConfig.getBranch(), authorConfig.getBranch());` are redundant since `Assert.assertEquals(expectedConfig, authorConfig);` already verified them.